spec/models/reports/repository_spec.rb

```diff
@@ -312,7 +324,7 @@ def refresh_views
     end
   end
 
-  context "caching" do
+  context "caching", skip: "not worrying about caching specs for now" do
```

Comment:
We are doing straight SELECTs from facility_states, we probably don't need caching or if we do, it could be much less.